src/rubicon/objc/types.py

```diff
@@ -779,6 +779,12 @@ class NSSize(Structure):
         ("height", CGFloat),
     ]
 
+    def __repr__(self):
+        return f"<NSSize({self.width}x{self.height})>"
```

Comment:
The brackets aren't really needed here - they're common when describing a point, but not when describing a size.
```suggestion
        return f"<NSSize {self.width}x{self.height}>"
```